src/plugins/defaultStatusForever/index.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+interface Choice {
+    key: number;
+    value: any;
+    label: string;
+}
+
+export default definePlugin({
+    name: "DefaultStatusForever",
+    description: "Make statuses default to last forever",
+    authors: [Devs.ImLvna],
+
+    patches: [
+        {
+            // hardcode default status duration to null
+            find: "this.clearAfterOptions",
+            replacement: {
+                match: /(?<=value:)\i(?=,options:this.clearAfterOptions)/,
+                replace: "null"
+            }
+        },
+        {
+            // reorder the list to put "Dont't Clear" at the top
+            find: "get clearAfterOptions",
+            replacement: {
+                match: /(?<=get clearAfterOptions\(\){return).*?}]/,
+                replace: " $self.patchChoices($&)"
+            }
+        }
```

Comment:
this is the same module, no?
```suggestion
        {
            find: "this.clearAfterOptions",
            replacement: [
            	// change default status duration to null
            	{
                	match: /(?<=value:)\i(?=,options:this.clearAfterOptions)/,
                	replace: "null"
            	},
            	// reorder the list to put "Dont't Clear" at the top
				{
                	match: /(?<=get clearAfterOptions\(\){return).*?}]/,
                	replace: " $self.patchChoices($&)"
            	}
            ]
        }
```